<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>547</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>595</width>
    <height>295</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources/resource.qrc">
    <normaloff>:/icons/icons/icon.ico</normaloff>:/icons/icons/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="rightLabel">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="FileTabSelector" name="leftTabWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="accessibleDescription">
       <string>Left</string>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="buttons">
      <item>
       <widget class="QPushButton" name="F5">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>F5 Copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="F6">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>F6 Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="F7">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>F7 New Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="F8">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>F8 Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="F9">
        <property name="text">
         <string>F9 Rename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="F10">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>F10 Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="leftLabel">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="FileTabSelector" name="rightTabWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="accessibleDescription">
       <string>Right</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Fi&amp;les</string>
    </property>
    <addaction name="action_new_file"/>
    <addaction name="action_new_directory"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_show_hidden_files"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QuickAccessBar" name="quickBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionRename">
   <property name="text">
    <string>&amp;Rename</string>
   </property>
  </action>
  <action name="actionDelete_selected">
   <property name="text">
    <string>&amp;Delete selected</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>&amp;Cut </string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>C&amp;opy </string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionCopy_selected">
   <property name="text">
    <string>Cop&amp;y selected</string>
   </property>
  </action>
  <action name="actionMove_selected">
   <property name="text">
    <string>&amp;Move selected</string>
   </property>
  </action>
  <action name="action_show_hidden_files">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show hidden files</string>
   </property>
  </action>
  <action name="action_new_file">
   <property name="text">
    <string>New file</string>
   </property>
  </action>
  <action name="action_new_directory">
   <property name="text">
    <string>New folder</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileTabSelector</class>
   <extends>QTabWidget</extends>
   <header>FileTabSelector.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QuickAccessBar</class>
   <extends>QToolBar</extends>
   <header>QuickAccessBar.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources/resource.qrc"/>
  <include location="Resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
